Miniature energy gun para el Magistrado .

SagaStars opened the pull request:
- Agrega una miniature energy gun para el magistrado en el locker de su oficina.

   Oh yeah mr krabs. Este PR ubicara un miniature energy gun personalizado para el magistrado para que tenga una pequeña proteccion extra por tener una ID con accesos a **Equipamiento de Sec** lo cual lo haria mas grande como objetivo, la miniature energy gun estara ubicada en su locker. 

_Imagen del sprite_
![Peter_Griffin](https://user-images.githubusercontent.com/55407528/65846315-58794a00-e30b-11e9-82e8-c8478bab3225.png)


_Changelog_
🆑 
add: objeto de miniature energy gun con sprite custom.
remove: codigo sucio y mugrento.
add: codigo chad y limpio. 
🆑

SagaStars committed:
no

no

SagaStars commented:
OHSI ME TOCO EL PR 1**69**#

Evankhell561 commented on code/HISPANIA/game/hispania_items.dm:
```diff
@@ -344,3 +344,29 @@
 
+///MINIATURE ENERGY GUN STARTS HERE///
+
+/obj/item/gun/energy/gun/minimagistrate
```
¿Ésta es una copia de la del hop, verdad? Solo tenias que hacer /obj/item/gun/energy/gun/mini/magistrate y añadir las variables name, desc, icon_state, e icon. Y con eso ya tienes tu pistola de magistrado. No necesitas copiar todo el codigo de la pistola del hop.

SagaStars committed:
Limpieza de codigo

gracias evan qliao.

-cambio de ruta de objetos para tener una replica que solo cambie la sprite y la desc.
-nueva ruta en el locker
-un ligero toquetillo al horrible "miniature energy gun"

SagaStars committed:
cambio al sprite

- literalmente eso, cambio a la skin

Evankhell561 commented on code/game/objects/structures/crates_lockers/closets/secure/security.dm:
```diff
@@ -456,6 +456,7 @@
 
 /obj/structure/closet/secure_closet/magistrate/New()
 	..()
+	new /obj/item/gun/energy/gun/mini/magistrate(src)
```
Puedes meter esto en el closet del magistrado desde un nuevo dm en hispania, tal como lo hiciste con el closet del hos.

iRyzor commented on code/HISPANIA/game/hispania_items.dm:
```diff
@@ -344,3 +344,13 @@
+
+/obj/item/gun/energy/gun/mini/magistrate
+	name = "miniature energy gun"
+	desc = "A small, pistol-sized energy gun with a built-in flashlight. It has fancy colors!."
```
Esa coma va ahí? :thinking:

iRyzor reviewed (commented):
1- Arregla el changelog.
2- Los sprites de armas tienen un sitio específico donde ponerlos.
3- Las armas tienen una carpeta específica donde ponerlos, no lo metas dentro de hispania_items.
4- El nombre del icon_state es un poco... genérico, sugiero que le pongas un nombre acorde al objeto.
5- Lo que dijo Evankhell.

SagaStars commented:
WHooPSIEEE DOOOOPSUS

![](https://cdn.discordapp.com/attachments/387645048386486273/645796715722571805/7130000c6a1bd6d935835712fe68482f.png)

![](https://cdn.discordapp.com/attachments/387645048386486273/645796931477569537/5ca4d22788e5f0b06820f043fbfe98d9.png)
